### PR TITLE
Updated the country description for Iraq in countries_names.txt

### PR DESCRIPTION
**Pull request by Colonel-Dixie**
Updated the country description for Iraq.

### CODE REVIEW

File: data/countries_names.txt [hunk 1]
```diff
@@ -52682,49 +52682,78 @@
...
+  pt = Mossul, Erbil, Kirkuk, Tikrit
+  ro = Mosul, Erbil, Kirkuk, Tikrit
+  ru = Мосул, Эрбиль, Киркук, Тикрит
+  sk = Mosul, Erbíl, Kirkúk, Tikrít
+  sv = Mosul, Erbil, Kirkuk, Tikrit
+  th = โมซูล, เออร์บิล, เคอร์คุก, ติกริต
+  tr = Musul, Erbil, Kerkük, Tikrit
+  uk = Мосул, Ербіль, Кіркук, Тікріт
+  vi = Mosul, Erbil, Kirkuk, Tikrit
+  bg = Мосул, Ербил, Киркук, Тикрит
```
Comment (biodranik) [inline data/countries_names.txt:52712]: Thanks! Please sort these translations by name (en should be the first one), and add
```
be = Масул, Эрбіл, Кірук, Цікрыт
```

File: data/countries_names.txt [hunk 2]
```diff
@@ -52682,49 +52682,78 @@
...
-  ru = Багдад, Кербела, Насирия
-  sk = Bagdad
-  sv = Bagdad
-  th = แบกแดด
-  tr = Bağdat
-  uk = Багдад, Кербела, Насирія
-  vi = Bagdad
+  en = Baghdad, Basra, Hillah, Ramadi
+  ar = بغداد، البصرة، الحلة، الرمادي
+  az = Bağdad, Bəsrə, Hillə, Ramadi
```
Comment (biodranik) [inline data/countries_names.txt:52724]: be = Багдад, Басра, Хіл, Рамадзі

File: data/countries_names.txt (hunk 1 above)
Comment (Colonel-Dixie) [reply data/countries_names.txt:52712]: Added.

File: data/countries_names.txt (hunk 2 above)
Comment (Colonel-Dixie) [reply data/countries_names.txt:52724]: Added

File: data/countries_names.txt [hunk 3]
```diff
@@ -52682,49 +52682,80 @@
...
+  uk = Мосул, Ербіль, Кіркук, Тікріт
+  vi = Mosul, Erbil, Kirkuk, Tikrit
+  bg = Мосул, Ербил, Киркук, Тикрит
+  hr = Mosul, Erbil, Kirkuk, Tikrit
+  lt = Mosulas, Erbilas, Kirkukas, Tikritas
+  lv = Mosula, Erbīla, Kirkūka, Tikrita
+  no = Mosul, Erbil, Kirkuk, Tikrit
+  sr = Мосул, Ербил, Киркук, Тикрит
+  sl = Mosul, Erbil, Kirkuk, Tikrit
+  zh = 摩苏尔, 埃尔比勒, 基尔库克, 提克里特
```
Comment (biodranik) [inline data/countries_names.txt:52720]: This should be split into zh-Hans for simplified Chinese and zh-Hant for traditional Chinese here and below.

Can you please also sort lines by alphabet (en is the first)?

Comment (Colonel-Dixie) [reply data/countries_names.txt:52720]: My apologies for this mistake and thank you for leading me to correct my mistake. I did correct the simplified and the traditional ones, and English was already in top of the languages who follow alphabetically in order.

Comment (biodranik) [reply data/countries_names.txt:52720]: No problem. Now the only change left is to sort all lines in each section by alphabet.

Comment (Colonel-Dixie) [reply data/countries_names.txt:52720]: > Now the only change left is to sort all lines in each section by alphabet.      
  
I sorted them by population, from the largest to the smallest. In the Iraq South section, the sorting happens to be alphabetical as well, but that was not intentional. In the Iraq North section, the cities are sorted only by size and population.

I don’t think moving Erbil and Kirkuk ahead of Mosul is appropriate, as Mosul is the second-largest city in Iraq (after Baghdad) and larger than those two.

So currently, Iraq South is sorted from the largest cities to the smallest, which also happens to be alphabetical, while Iraq North is sorted only by size and population.

Comment (biodranik) [reply data/countries_names.txt:52720]: @Colonel-Dixie sorry, maybe I wasn't clear. The translations order of all other languages should be sorted:
1. en
2. af
3. ar
4. ...

File: data/countries_names.txt [hunk 4]
```diff
@@ -52682,49 +52682,82 @@
...
+  vi = Mosul, Erbil, Kirkuk, Tikrit
+  bg = Мосул, Ербил, Киркук, Тикрит
+  hr = Mosul, Erbil, Kirkuk, Tikrit
+  lt = Mosulas, Erbilas, Kirkukas, Tikritas
+  lv = Mosula, Erbīla, Kirkūka, Tikrita
+  no = Mosul, Erbil, Kirkuk, Tikrit
+  sr = Мосул, Ербил, Киркук, Тикрит
+  sl = Mosul, Erbil, Kirkuk, Tikrit
+  zh-hans = 摩苏尔, 埃尔比勒, 基尔库克, 提克里特
+  zh-hant = 摩蘇爾, 埃爾比勒, 基爾庫克, 提克里特
```
Comment (biodranik) [inline data/countries_names.txt:52721]: 1. It should be zh-Hans and zh-Hant (capital H)
2. Translations should be sorted, here is the validation check that fails if translations are not sorted: https://github.com/organicmaps/organicmaps/actions/runs/9798311426/job/27056526538

Comment (Colonel-Dixie) [reply data/countries_names.txt:52721]: My apologies. I just edit it and made the required changes to fix the errors.
zh-hans and zh-hant to zh-Hans and zh-Hant.     
And order the languages alphabetically, the  entries (bg, hr, lt, lv, no, sr, sl, zh-Hans, zh-Hant) were not in alphabetical order before.    
 https://github.com/organicmaps/organicmaps/actions/runs/9911301077/job/27383649042?pr=8626

**Comment by Colonel-Dixie** [comment]
Wow, rookie mistake by me, closed by a mistake, did not thought that deleting the fork would delete the pull request. Will send another pull request. Sorry for that.

**Pull request closed without merging**